### PR TITLE
[enchanment](udf) add more info when download jar package failed

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes
when download jar package, some times show the checksum is not equal,
but the root reason is unknown, now add some error msg if failed.

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run p0

**Comment by zhangstar333** [comment]
run buildall

File: be/src/runtime/user_function_cache.cpp [hunk 1]
```diff
@@ -271,10 +285,13 @@ Status UserFunctionCache::_download_lib(const std::string& url,
...
     Md5Digest digest;
     HttpClient client;
+    int64_t file_size;
     RETURN_IF_ERROR(client.init(real_url));
     Status status;
-    auto download_cb = [&status, &tmp_file, &fp, &digest](const void* data, size_t length) {
+    auto download_cb = [&status, &tmp_file, &fp, &digest, &file_size](const void* data,
+                                                                      size_t length) {
         digest.update(data, length);
+        file_size = file_size + length;
```
Comment (morningman) [inline be/src/runtime/user_function_cache.cpp:294]: what is the init value of `file_size`?

**Comment by zhangstar333** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**